libs/s25main/desktops/dskCredits.cpp

```diff
@@ -308,38 +322,33 @@ void dskCredits::Draw_()
 
 void dskCredits::GotoNextPage()
 {
-    ++this->itCurEntry;
-    if(this->itCurEntry == entries.end())
-        this->itCurEntry = entries.begin();
-    timer.restart();
+    ++itCurEntry;
```

Comment:
we probably should snake-case that var here for consistency - like `it_cur_entry_`

Reply:
or at least add the `_` to all members

Reply:
Erm, everything is camelCase

Reply:
ah yes we have different styles everywhere :-D